apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-intent/use-site-intent.js

```diff
@@ -2,17 +2,24 @@ import apiFetch from '@wordpress/api-fetch';
 import { useState, useEffect, useCallback } from '@wordpress/element';
 
 const useSiteIntent = () => {
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
FYI
I found another useSiteIntent which can be used from `'@automattic/data-stores'`
https://github.com/Automattic/wp-calypso/blob/fbb9ff01021354bb613d31281510386cd806485c/packages/data-stores/src/queries/use-site-intent.ts#L4

Reply:
I guess the reason is we don't introduce `react-query` to all ETK plugins at least for the `wpcom-block-editor-nux`. We can create follow-up PRs to replace `useSiteIntent` with the one from `@automattic/data-stores`

Reply:
Thank you for letting me know!
For now, I added a comment to replace this `useSiteIntent`. https://github.com/Automattic/wp-calypso/pull/73565/commits/1e42acdfc8cd9b10c1e91d4763ac408e2c8220c4